django_slack/api.py

```diff
@@ -1,5 +1,6 @@
 import json
 
+import six
```

Comment:
no problem, but the blank line was added intentionally because of pep-8 rules for imports:
```Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.```